### ISSUE TITLE
conservation measures: create codelist and conservationMeasures object within new environment object

### ISSUE THREAD

**Issue by odscjen** [open-contracting/infrastructure#401]
# Background

This issue relates to the following CoST IDS elements proposed in the CoST IDS/OC4IDS review:

<details>

<summary> Conservation measures </summary>

## Conservation measures

**Module:** Environmental and climate
**Indicator:** Biodiversity impact

### Disclosure format

> Disclose and provide further details on the measures adopted by the project to protect and enhance biodiversity. This can comprise, without limitation the following:   
​​avoidance of ecological sitting 
buffers for ecological land 
nature-based solutions 
land restoration 
protection to landscape and historical sites 
invasive species management 
management of wildlife mortality risk 
reduce habitat loss 
pollution reduction 
soil management 
hazardous material management]) 
others (explain)

### OC4IDS mapping

>Project level: For each measure add a `ConservationMeasure` object to the `environment.conservationMeasures` array, setting its `.type` according to the matching code in the open “conservationMeasuresType” codelist and mapping the further explanation to its `.description`. If there is an accompanying document publish in `documents`, with `.documentType` set to 'conservationMeasures’ and include a short description and/or a link to a document providing details. If the details are part of a more general document indicate the relevant section of the document using `.pageStart` and `.pageEnd`. 

</details>

# Proposal

Create a `conservationMeasure` object comprising a `type` and `description`. An array of these measures will be created as `conservationMeasures` within the new `environment` object (see https://github.com/open-contracting/infrastructure/issues/393). `conservationMeasure.type` will take from an open codelist, conservationMeasures.

“conservationMeasures” will be added to the documentType codelist to identify accompanying documentation the publisher may have.

Research is still required to fully define the conservationMeasures codelist.

Add the following fields and objects:

Path | Title | Description | Type (format) | Codelist
-- | -- | -- | -- | --
environment.conservationMeasures | Conservation measures | The measures adopted by the project to protect and enhance biodiversity. | array of `ConservationMeasures` objects | -
ConservationMeasure | Conservation measure | A measure adopted by the project to protect and enhance biodiversity. | object |  
ConservationMeasure.type | Type | The type of conservation measure, using the open conservationMeasures codelist. | string | conservationMeasures
ConservationMeasure.description | Description | A short description of the conservation measure. |   |  

</div>

Add the following codes:

Codelist | Code | Title | Description
-- | -- | -- | --
conservationMeasures | TBD | Avoidance of ecological setting | TBD
conservationMeasures | TBD | Buffers for ecological land | TBD
conservationMeasures | TBD | nature based solutions | TBD
conservationMeasures | TBD | land restoration | TBD
conservationMeasures | TBD | protection to landscape and historical sites | TBD
conservationMeasures | TBD | invasive species management | TBD
conservationMeasures | TBD | management of wildlife mortality risk | TBD
conservationMeasures | TBD | reduce habitat loss | TBD
conservationMeasures | TBD | pollution reduction | TBD
conservationMeasures | TBD | soil management | TBD
conservationMeasures | TBD | hazardous material management | TBD
documentType | conservationMeasures | Conservation measures | Documentation detailing measures to be taken to mitigate the project’s impact on biodiversity

</div>

# Example

```json

{ 
 "environment": { 
   "conservationMeasures": [ 
 	{ 
   	"type": "landRestoration", 
   	"description": "Land restoration measures for the project include..." 
 	} 
   ] 
 }, 
 "documents": [ 
   { 
 	"id": "1", 
 	"documentType": "conservationMeasures", 
 	"url": "http://example.com/conservationMeasures.pdf" 
   } 
 ] 
} 

```

# Sources

* [WWF infrastructure - State of the practice report](https://wwfint.awsassets.panda.org/downloads/wwf_infrastructure_full_report_2018.pdf)
* [WWF KPMG Measuring sustainability in infrastructure investment](https://wwfeu.awsassets.panda.org/downloads/kpmg_full_report.pdf)
* [Testing industry attitudes toward a common reporting approach for ESG data use in infrastructure investment](https://www.guggenheiminvestments.com/GuggenheimInvestments/media/PDF/2022-KPMG-ESG-Data-Report.pdf)
* [Key assessment indicators for teh sustainability of infrastructure projects](https://www.researchgate.net/publication/265623124_Key_Assessment_Indicators_for_the_Sustainability_of_Infrastructure_Projects)
* [Playbook for sustainable infrastructure - WWF](https://issuu.com/fidic/docs/wwf_playbook_-_single_pages?fr=xKAE9_zU1NQ)

cc @evelyndinora

**Comment by odscjen** [comment]
Note from Maria at CoST:
> "A [new FIDIC/WWF report published in September](https://issuu.com/fidic/docs/wwf_playbook_-_single_pages?fr=xKAE9_zU1NQ) can help us create categories of conservation measures. Here are key categories they used in case it helps - they are mostly associated with nature-based solutions:
> * Afforestation, reforestation and woodland creation
> * Open green spaces and tree planting
> * Green corridor and green walls and roofs
> * Reintroduction of species
> * Creation of new protected area
> * Revegetation of slopes
> * Landscape and river restoration 
> * Catch management
> * Coastal restoration and connected measures (ecological seawalls, restoration of mangrove forests, salt marsh, coral reefs, wetlands and sea grass)
> * Marine life and marine habitat restoration
> * Economic instruments for biodiversity and habitat offsetting 
> * Erosion protection
> * Sludge treatment
> * Pollution treatment
> * Wetlands (restoration and creation)
>* Use of low-impact materials"

There's little overlap between this list and the originally suggested list of terms, although as Maria says a number of these new terms could all come under "nature based solutions".

**Comment by odscjen** [comment]
Looking through the [IFC Performance Standard 6: Biodiversity conservation and sustainable management of living natural resources](https://www.ifc.org/content/dam/ifc/doc/2010/2012-ifc-performance-standard-6-en.pdf) they mention the following conversation measures that a project may take:
* biodiversity offsets ("which may be considered only after appropriate avoidance, minimization, and restoration measures have been applied." para 10) where a biodiversity offset is defined as "measurable conservation outcomes resulting from actions designed to compensate for significant residual adverse biodiversity impacts arising from project development and persisting after appropriate avoidance, minimization and restoration measures have been taken."
* Identification and protection of set-asides (defined as "land areas within the projects site, or areas over which the client has management control, that are excluded from development and are targeted for the implementation of conservation enhancement measures." (footnote 10, para 15)
* Biological corridors
* Habitat restoration during and/or after operations
* Avoidance of "potential for accidental or unintended introductions ... that may harbor alien species." (para 22)
* "Conduct systemic review to identify priority ecosystem services"
* Application of industry specific good land management practices
* "Implement sustainable management practices to one of more relevant and credible standards as demonstrated by independent verification or certification." (para 26)
* "limit procurement to those suppliers that can demonstrate that they are not contributing to significant conversion of natural and/or critical habitats" (para 30)

**Comment by odscjen** [comment]
The [SuRe - Standard for Sustainable and resilient infrastructure](https://sure-standard.org/wp-content/uploads/2021/07/ST01_Normative_Standard_v2.0_clean.pdf) includes "Theme E2: Biodiversity and Ecosystems" that covers related requirements of infrastructure projects that wish to be certified by this body. There are few specific measures to be taken detailed in this standard, it focuses more on the required outcome and associated evidence to be provided. But broad measures mentioned include:
* biodiversity offsets - based on the Business and Biodiversity Offsets Programme concepts and methodologies and using the same definition as the IFC report in the previous comment
* consultation with Affected Communities
* "execute construction alternatives that do not interfere with ecological corridors" (E2.2 A.4, pg 120)
* execute nature based solutions whenever beneficial
* "prevent the deliberate or accidental introduction of alien or non-native species" (E2.3 A, pg 122)
* combat invasive species within project boundaries, focussing on biological and biotechnical measures"

Theme E4: Pollution
* "The project...should integrate sustainable soil management technicques according to Food and Agriculture Organization (FAO's) Charter and Sustainable Soil Management Criteria" (E4.1 A.2 pg 136)
* "The Project shall maintain air pollutant emissions below those specified by: applicable local regulations or guidelines; the World Health Organization (WHO) Air Quality Guidelines; and those specified within the World Bank Group (WBG) Environmental, Health, and Safety (EHS) Guidelines; whichever is more stringent" (E4.1 A.4 pg 136)
* "restoring or rehabilitating nearby degraded soils" (E4.1 D pg 136)
* "restoring nearby water ecosystems" (E4.2 D, pg 138)
* user of integrated pest management

Theme E5: Land use and Landscape
* location analysis to pick a site that minimises negative impacts associated with construction and operation including minimising use of green-field land.

**Comment by odscjen** [comment]
[BREEAM Infrastructure projects international version](https://bregroup.com/products/ceequal/the-ceequal-technical-manuals/#manual-download) section 4: Land use and ecology. As with SuRe this standard is focused more on the outcomes and evidence than the specific measures but contains the following mentions of measures:
* The project siting took into consideration alternative sites to minimise use of undeveloped land
* land remediation (multiple processes and technologies, each with different time scales and outcomes)
* enhancing existing ecological features
* creation of new wildlife habitats, on or off-site
* install special structures or facilities for wildlife
* improving water environment (e.g. introducing aquatic plants to help cleanse the water, removing invasive aquatic plants, capturing rain water for beneficial use)

Section 6: Pollution
* prevention of leakage of pollutants into a watercourse or the sea
* mitigation of noise, light, emissions and vibrational nuisance pollution during construction (as applicable to human and non-human neighbours of the project)

**Comment by odscjen** [comment]
Based on the research quoted above I'm suggesting we convert the initial suggested data point list into the following codes:

Codelist | Code | Title | Description
-- | -- | -- | --
conservationMeasures | locationAnalysis | Location analysis | Location analysis was performed to select the site with the minimum potentially negative ecological impacts as a result of the project.
conservationMeasures | biologicalCorridors | Biological corridors | Areas of habitat connecting wildlife populations, or different habitat types necessary for species lifecycles, separated by human activities or structures. Also known as wildlife or green corridors.
conservationMeasures | natureBasedSolutions | Nature based solutions | Conservation measures that use solutions already found in nature, for example conserving mangrove forests to protect homes against storm damage.
conservationMeasures | habitatRestoration | Habitat restoration | The restoration of ecosystems negatively impacted by human activities, both historically and as a direct result of the project.
conservationMeasures | setAsides | Set-asides | Identification of areas within the project boundaries that are excluded from development and targeted for conservation enhancement measures.
conservationMeasures | invasiveSpeciesManagement | Invasive species management | Measures taken to prevent the introduction of alien, invasive or non-native species, and combat the presence of such species identified within the project boundaries.
conservationMeasures | habitatLossReduction | Reduction of loss of habitat | Measures that reduce the ecological habitats lost as a direct result of the project.
conservationMeasures | pollutionReduction | Reduction of pollution | Measures that reduce the level of pollution generated as a direct result of the project. Pollution can include leakage into water courses or soil as well as noise, light, aerosol and gaseous emissions or vibrational nuisance to nearby wildlife populations.
conservationMeasures | soilManagement | Soil management | Measures that integrate sustainable soil management practices according to the Food and Agriculture Organization’s Charter and Sustainable Soil Management Criteria.
conservationMeasures | biodiversityOffsets | Biodiversity offsets | Measurable outcomes as a result of actions designed to compensate for adverse biological impacts due to the project that persist after avoidance, minimization and restoration measures have been taken.
documentType | conservationMeasures | Conservation measures | Documentation detailing measures to be taken to mitigate the project’s impact on biodiversity

It makes sense to keep these fairly high level and as there is the associated `.description` field that publisher can use to give fuller details, e.g.

```json
{ 
 "environment": { 
   "conservationMeasures": [ 
 	{ 
   	"type": "natureBasedSolutions", 
   	"description": "Green walls and roofs shall be incorporated into the buildings design" 
 	} 
   ] 
 }
}
```

Reasoning for removing some of the originally suggested points:
* protection to landscape and historical sites - I don’t think “historical sites” falls under the definition of this data point which is about measures adopted to “protect and enhance biodiversity”. Protection of landscape is too broad and covered by other codes namely habitatRestoration, setAsides and ecologicalSitingAnalysis
* management of wildlife mortality risk - I didn’t find this mentioned in any of the other standards so it’s unclear what these measures could be that aren’t already covered by other codes namely set-Asides, biologicalCorridors, ecologicalSitingAnalysis and habitatLossReduction
* hazardous material management - again I don’t think this fits the definition of this data point and the specific elements that could fall under this as being about enhancing and protecting biodiversity are covered by pollutionReduction.

cc @mgraca-prado, @EvelynDinora

**Comment by odscjen** [comment]
@mgraca-prado, @EvelynDinora @jpmckinney any thoughts, objections of clarifications needed for the proposed object in the issue description or the codelist codes and descriptions as given in [this comment](https://github.com/open-contracting/infrastructure/issues/401#issuecomment-1785145340)

**Comment by jpmckinney** [comment]
LTGM, except I don't understand the combination of

* ecologicalSitingAnalysis ("siting"?)
* Avoidance of ecological setting (not sure what this means)
* Location analysis was performed to select a site to minimize potentially negative ecological impacts as a result of the project.

Can you share quotes from sources for this code?

**Comment by odscjen** [comment]
Yeah that was a really tricky one that I probably didn't get quite right. The original code suggested by CoST was "​​avoidance of ecological sitting". From my research the most relevant metric I found came from SuRe:

> "The Project shall carefully assess the Project location and siting during the design and construction phases of the Project, in consultation with Affected Communities and relevant public authorities. The Project shall also avoid degrading critical Natural Capital areas that provide important Ecosystem Services (at various scales - both local and global)."

So it's about saying that the project conducted analysis to try and ensure that out of the potential locations available, the location/site they picked minimized the potential for ecological damage, i.e. avoided picking a site with high ecological value.

Perhaps a better version would be:

Code: locationAnalysis
Title: Location analysis
Description: Location analysis was performed to select the site with the minimum potentially negative ecological impacts as a result of the project.

As it's in a codelist called conservationMeasures the code probably doesn't need to explicitly mention ecology

**Comment by EvelynDinora** [comment]
@odscjen the description sounds good to us, many thanks!

**Comment by odscjen** [comment]
@jpmckinney does that new version of the code make more sense?

**Comment by jpmckinney** [comment]
LGTM
